common/src/main/java/io/github/axolotlclient/modules/hud/gui0/hud/simple/ReachHud.java

```diff
@@ -20,62 +20,60 @@
  * For more information, see the LICENSE file.
  */
 
-package io.github.axolotlclient.modules.hud.gui.hud.simple;
+package io.github.axolotlclient.modules.hud.gui0.hud.simple;
 
+import io.github.axolotlclient.AxolotlClientConfig.api.options.Option;
+import io.github.axolotlclient.AxolotlClientConfig.impl.options.IntegerOption;
+import io.github.axolotlclient.bridge.AxoMinecraftClient;
+import io.github.axolotlclient.bridge.Platform;
+import io.github.axolotlclient.bridge.entity.AxoEntity;
+import io.github.axolotlclient.bridge.events.Events;
+import io.github.axolotlclient.bridge.math.Vec3;
+import io.github.axolotlclient.bridge.util.AxoI18n;
+import io.github.axolotlclient.bridge.util.AxoIdentifier;
+import io.github.axolotlclient.modules.hud.gui0.entry.SimpleTextHudEntry;
 import java.math.RoundingMode;
 import java.text.DecimalFormat;
 import java.util.List;
-
-import io.github.axolotlclient.AxolotlClientConfig.api.options.Option;
-import io.github.axolotlclient.AxolotlClientConfig.impl.options.IntegerOption;
-import io.github.axolotlclient.modules.hud.gui.entry.SimpleTextHudEntry;
-import net.minecraft.client.Minecraft;
-import net.minecraft.client.resource.language.I18n;
-import net.minecraft.entity.Entity;
-import net.minecraft.resource.Identifier;
-import net.minecraft.util.math.Vec3d;
+import java.util.Objects;
 
 // https://github.com/AxolotlClient/AxolotlClient-mod/blob/4ae2678bfe9e0908be1a7a34e61e689c8005ae0a/src/main/java/io/github/axolotlclient/modules/hud/gui/hud/ReachDisplayHud.java
 // https://github.com/DarkKronicle/KronHUD/blob/703b87a7c938ba25da9105d731b70d3bc66efd1e/src/main/java/io/github/darkkronicle/kronhud/gui/hud/simple/ReachHud.java
 public class ReachHud extends SimpleTextHudEntry {
 
-	public static final Identifier ID = new Identifier("kronhud", "reachhud");
+	public static final AxoIdentifier ID = AxoIdentifier.of("kronhud", "reachhud");
 	private final IntegerOption decimalPlaces = new IntegerOption("decimalplaces", 0, 0, 15);
 
 	private String currentDist;
 	private long lastTime = 0;
 
-	public static double getAttackDistance(Entity attacking, Entity receiving) {
-
-		Vec3d camera = attacking.getRotationVec(1);
-		return camera.distanceTo(receiving.getRotationVec(1));
+	private static double getAttackDistance(AxoEntity attacking, AxoEntity receiving) {
+		Vec3 camera = attacking.getRotation(1);
+		return camera.dist(receiving.getRotation(1));
```

Comment:
Other versions do a manual raycast here which *probably* works better. However, the method used is not available on 1.8.9.